spec/instant.html

```diff
@@ -26,7 +26,7 @@ <h1>The Temporal.Instant Constructor</h1>
     <emu-clause id="sec-temporal.instant">
       <h1>Temporal.Instant ( _epochNanoseconds_ )</h1>
       <p>
-        When the `Temporal.Instant` function is called, the following steps are taken:
+        The `Temporal.Instant` function performs the following steps when called:
```

Comment:
Are these changes really helpful? It seems like switching to structured headers (in which such raw spec text just goes away) would be better.

Reply:
There are no structured headers for functions yet, only for abstract operations.